tests/foreman/api/test_host.py

```diff
@@ -1387,6 +1387,31 @@ def test_positive_read_puppet_ca_proxy_name(
     assert session_puppet_enabled_proxy.name == host['puppet_ca_proxy_name']
 
 
+@pytest.mark.tier2
+def test_positive_list_hosts_thin_all(module_target_sat):
+    """List hosts with thin=true and per_page=all
+
+    :id: 00b7e603-aed5-4b19-bfec-1a179fad6743
+
+    :expectedresults: Hosts listed without ISE
+
+    :BZ: 1969263
+
+    :customerscenario: true
+    """
+    hosts = module_target_sat.api.Host().search(query={'thin': 'true', 'per_page': 'all'})
+    assert module_target_sat.hostname in [host.name for host in hosts]
```

Comment:
I wonder why the satellite host itself comes under `thin` provisioned host. Maybe someone from DEVs can answer but I think if we could create a dummy thin host that would be quicker as well as a robust verification.

Reply:
This has nothing to do with thin **provisioning**.

Reply:
Yeah not exactly `provisioning` but thin relates to how we allocate the storage to the VM. Even my comment is not focused on provisioning but on how we are testing it.

Reply:
No. Thin in this context means something completely different. A thin list of hosts is a list of hosts that only contains ID and name of the host.